README.md

```diff
@@ -70,24 +70,62 @@ for these new features.
 #### SSO Clients and LARA (authoring) integration
 
 These instructions assume that you are setting up LARA and Portal using
-docker-compose files. It *may* also assume that you are using something like Dori or Dinghy
-as an http-proxy dns container, so that the portal is available at:`//app.portal.docker`
-and lara is available at `//app.lara.docker`
+docker-compose files. It also assumes that you are using something like Dori or Dinghy
+as an http-proxy dns container, so that the portal is available at `app.portal.docker`
+and lara is available at `app.lara.docker`
 
-If you want to provide authentication services to LARA, you need to:
-1. In the Portal, edit `.env` and append `docker/dev/docker-compose-lara-proxy.yml` to the `COMPOSE_FILE` var.
-1. In the Portal, as an administrator, setup a new "Auth Client". Use the following settings:
+##### Starting from scratch on a Mac
+
```

Comment:
My 2 cents:
1. At some point during the setup, we need to manually run 
`bundle config gems.railslts.com username:password `
with the correct username and password. I think it's after you actually hit docker-compose up, but I might be wrong. It might make sense to add it here.
2. Also, both the times I setup my local environment from scratch, I had to run : 
`docker-compose run app bundle exec rake db:setup`
Otherwise it just wouldn't find the portal_development db.
3. We might also want to add a step to add the firebase app since I guess without it, student answers don't save, reports don't work, etc,.
4. Should we also add the portal certificate settings instructions from the lara repo here for the sake of completeness?

Reply:
Yeah I wasn't sure how to deal with this. There is 2 other places with docker setup documentation in the Portal. So for now I just stuck with the simplified instructions in this section. 
Neither of those other docker sections address points 1,2, and 3 though. So it would be best to update them, and then change this section to refer to those sections to setup the portal first.

In the meantime I'll put a comment in steps 2 and 4 of the "starting from scratch" subsection, that this is in complete and oversimplified.

I do hope this will be streamlined with rake tasks soon since we'll need those to automate the setup of a Portal and LARA instances for the ephemeral services to test PRs.

Reply:
I'd note that at least in theory if you are starting from a clean checkout. This code should automatically run `db:setup` and also run the `sso:add_dev_client`.
https://github.com/concord-consortium/rigse/blob/1d557c057a1bb63cdcd167c87957f8d16fe78be8/rails/docker/dev/run.sh#L34-L39